### ISSUE TITLE
Table of contents control missing visible focus indicator

### ISSUE THREAD

**Issue by dbjorge** [a11yproject/a11yproject.com#1173]
## Summary

The collapsible "Table of contents" element used in the left nav doesn't have a visible focus indicator (required by [WCAG SC 2.4.7 Focus Visible](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html)). The rest of the table of contents uses a pink line to the right of each entry to indicate focus, but the focused state for the "Table of contents" control itself is not distinguishable from its unfocused state:

![screenshot of the Table of contents element](https://user-images.githubusercontent.com/376284/103025219-58bcc100-451f-11eb-96d1-104002f0783a.png)

### Expected result

Control should have some visible indication of being focused

### Actual result

Focused/unfocused states look the same

### Additional Information

* Verified on Windows on desktop Edge stable (87.0.664.66) and Firefox stable (84.0.1).
* Issue repros identically on [posts](https://www.a11yproject.com/posts/), [checklist](https://www.a11yproject.com/checklist/), [resources](https://www.a11yproject.com/resources/), and [about](https://www.a11yproject.com/about/) pages
* No ATs in use

### Additional Information

shameless plug: consider running the major pages of the site through an [Accessibility Insights for Web](https://accessibilityinsights.io/) assessment; it includes a test for this class of issue (3.1 Visible Focus)

**Comment by ericwbailey** [comment]
Great catch, thank you! Closed via https://github.com/a11yproject/a11yproject.com/pull/1174.
